packages/frontend/pages/index.vue

```diff
@@ -1,3 +1,3 @@
 <template>
-  <p>Hello world</p>
+  <LoginView/>
```

Comment:
`/` !== `/login`, I'll add redirect to `/login` in the future, but we don't want to use `/` as a login page.